config/main.json

```diff
@@ -9,7 +9,7 @@
     "polkadot": "v0.17.5",
     "prometheus-node-exporter": "1.11.1",
     "prometheus-operator": "9.3.1",
-    "substrate-telemetry": "v1.1.0",
+    "substrate-telemetry": "v1.1.1",
```

Comment:
Does this chart version exist ?

Reply:
We should use the last one, even if it exists it would not fix the issue with 0 peers reported.

Reply:
you mean the v2.4.1 , isn't it ?
Do you expect any compatibility issue ?

Reply:
should be there, https://github.com/w3f/substrate-telemetry-chart/releases/tag/v1.1.0

Reply:
maybe there are some changes required bc of how the bind host is declared now, https://github.com/w3f/substrate-telemetry-chart/commits/v2.0.0 something that should not affect us in the kusama deployment made by polkadot-deployer. 

In any case this change won't break anything about the validators, more about how their metrics are retrieved, we just need to communicate properly before deploying and keep an eye on it.